source/release-notes.txt

```diff
@@ -10,6 +10,44 @@ Release Notes
    :depth: 1
    :class: twocols
 
+|compass| 1.37.0
+----------------
+
+*Released May 25, 2023*
+
+New Features:
+
+- Dark mode has been updated with a modern theme. If the modern theme 
+  is enabled in ``Settings>Feature Preview``, Compass defaults to 
+  the modern dark mode theme.
+- Various user experience improvements when using the Import and Export functionality in Compass (:issue:`COMPASS-6543`).
+- Combine array fields into one in the import CSV preview (:issue:`COMPASS-6766`).
+- Add settings to sidebar menus (:issue:`COMPASS-6796`).
+- Show password only when user focuses on input (:issue:`COMPASS-6161`).
+- Add autocomplete support for $percentile, $median and $$USER_ROLES (:issue:`COMPASS-6780`) (:issue:`COMPASS-6781`).
+- Toggle state of **Edit connection string** is based on the new global preference. This setting controls whether a password is visible when creating a new connection.
+- Compass supports the new Queryable Encryption protocol. Starting
+  in v1.37.0, Compass is not compatible with MongoDB server versions
+  **earlier than 7.0** when using Queryable Encryption (:issue:`COMPASS-6601`) (:issue:`COMPASS-6602`).
+
+  - When using queryable encryption on pre-7.0 servers, you can decrypt
+    encrypted data, but you cannot insert or query data.
+
+- Add export aggregation code preview to export modal (:issue:`COMPASS-6725`).
+
+Bug Fixes:
+
+- Differentiate between new Date() and Date() (:issue:`COMPASS-6755`).
+- Fix guessFileType() when JSON fails and CSV lines are huge.
+- Flush import progress throttle on import error.
+- Feature flag default values (:issue:`COMPASS-6525`).
+- Allow updates on a sharded collection (:issue:`COMPASS-6058`).
+- Detect line breaks, pass it on to papaparse (:issue:`COMPASS-6819`).
+- Update reset on query bar to reset results and emit query-changed (:issue:`COMPASS-6805`).
+
+`Full Changelog avaialble on GitHub
```

Comment:
```suggestion
`Full Changelog available on GitHub
```

Reply:
Looks like this typo is also in the `1.36.3` release if we want to fix that as well.

Reply:
That makes sense, I copied and pasted from there.